2021-02-16 15:47:45,633 INFO    MainThread:12951 [wandb_setup.py:_flush():70] setting env: {}
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_init.py:_log_setup():319] Logging user logs to /home/naddeok5/FIM/wandb/run-20210216_154745-3age49kw/logs/debug.log
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_init.py:_log_setup():320] Logging internal logs to /home/naddeok5/FIM/wandb/run-20210216_154745-3age49kw/logs/debug-internal.log
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_init.py:init():324] calling init triggers
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_init.py:init():331] wandb.init called with sweep_config: {}
config: {'epochs': 1, 'batch_size': 128, 'momentum': 0.9, 'weight_decay': 0.005, 'learning_rate': 0.01, 'optimizer': 'sgd', 'criterion': 'mse'}
2021-02-16 15:47:45,634 INFO    MainThread:12951 [wandb_init.py:init():401] starting backend
2021-02-16 15:47:45,657 INFO    MainThread:12951 [backend.py:ensure_launched():81] starting backend process...
2021-02-16 15:47:45,676 INFO    MainThread:12951 [backend.py:ensure_launched():86] started backend process with pid: 13375
2021-02-16 15:47:45,677 INFO    MainThread:12951 [wandb_init.py:init():410] backend started and connected
2021-02-16 15:47:45,680 INFO    MainThread:12951 [wandb_init.py:init():433] updated telemetry
2021-02-16 15:47:45,681 INFO    MainThread:12951 [wandb_init.py:init():456] communicating current version
2021-02-16 15:47:46,530 INFO    MainThread:12951 [wandb_init.py:init():461] got version response upgrade_message: "wandb version 0.10.19 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-16 15:47:46,530 INFO    MainThread:12951 [wandb_init.py:init():469] communicating run to backend with 30 second timeout
2021-02-16 15:47:46,711 INFO    MainThread:12951 [wandb_init.py:init():492] starting run threads in backend
2021-02-16 15:47:48,091 INFO    MainThread:12951 [wandb_run.py:_console_start():1411] atexit reg
2021-02-16 15:47:48,091 INFO    MainThread:12951 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.REDIRECT
2021-02-16 15:47:48,093 INFO    MainThread:12951 [wandb_run.py:_redirect():1279] Redirecting console.
2021-02-16 15:47:48,093 INFO    MainThread:12951 [redirect.py:install():213] install start
2021-02-16 15:47:48,093 INFO    MainThread:12951 [redirect.py:install():228] install stop
2021-02-16 15:47:48,094 INFO    MainThread:12951 [redirect.py:install():213] install start
2021-02-16 15:47:48,094 INFO    MainThread:12951 [redirect.py:install():228] install stop
2021-02-16 15:47:48,094 INFO    MainThread:12951 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-16 15:47:48,094 INFO    MainThread:12951 [wandb_init.py:init():515] run started, returning control to user process
2021-02-16 15:47:55,978 INFO    MainThread:12951 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 1
2021-02-16 15:47:55,979 INFO    MainThread:12951 [wandb_run.py:_restore():1353] restore
2021-02-16 15:47:55,979 INFO    MainThread:12951 [redirect.py:uninstall():232] uninstall start
2021-02-16 15:47:55,980 INFO    MainThread:12951 [redirect.py:_stop():287] _stop: stdout
2021-02-16 15:47:55,981 INFO    MainThread:12951 [redirect.py:_stop():293] _stop closed: stdout
2021-02-16 15:47:55,981 INFO    stdout    :12951 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-16 15:47:55,982 INFO    stdout    :12951 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:_stop():299] _stop joined: stdout
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:uninstall():236] uninstall done
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:uninstall():232] uninstall start
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:_stop():287] _stop: stderr
2021-02-16 15:47:55,982 INFO    MainThread:12951 [redirect.py:_stop():293] _stop closed: stderr
2021-02-16 15:47:55,982 INFO    stderr    :12951 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-16 15:47:55,982 INFO    stderr    :12951 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-16 15:47:55,983 INFO    MainThread:12951 [redirect.py:_stop():299] _stop joined: stderr
2021-02-16 15:47:55,983 INFO    MainThread:12951 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-16 15:47:55,983 INFO    MainThread:12951 [redirect.py:uninstall():236] uninstall done
2021-02-16 15:47:56,123 INFO    MainThread:12951 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-02-16 15:47:58,129 INFO    MainThread:12951 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2021-02-16 15:47:59,491 INFO    MainThread:12951 [wandb_run.py:_show_files():1726] logging synced files
